### DISCUSSION TITLE
version 2.0.0 release date

### DISCUSSION THREAD

**Discussion by marklonquist** [SixLabors/ImageSharp#1878] [Q&A]
Hi

So the release for milestone version 2.0.0 has a past due for over a month. Can it be expected to get a new date?

We're eagerly waiting for webp support. :-)

**Comment by antonfirsov** [comment]
My personal estimation: sometime in January. Based on [backlog size](https://github.com/SixLabors/ImageSharp/milestone/8), I would rather say it's closer to the end of the month. Note that this is not a promise in any sense.

@JimBobSquarePants shall we update the milestone date to something realistic?

**Reply by JimBobSquarePants** [reply]
Yeah. I'll shift to 1st Feb. If we manage to ship early Yay!

**Reply by syunakh** [reply]
Hi
do you have any updates about release date?

**Reply by JimBobSquarePants** [reply]
This week hopefully. 1 more thing to look at and some docs to write

**Reply by antonfirsov** [reply]
I want to write conceptual docs, lots of them. Also create some examples for the blog post accompanying the release. #1739 is a beast that has to be explained to users.

I had some serious personal distractions the last month that prevented me focusing on this. Hopefully I will be able to start this work now, but I can't finish it this week. Maybe we can push out a `2.0-RC1` prerelease in the meanwhile, or just wait a bit more.

**Reply by JimBobSquarePants** [reply]
I've had a quick go at the pixel format docs. I don't think we have to justify our approach other than to say, this is how it works and point to the span caveats further down the page. Elsewhere I'm going to update any code samples and docs containing breaking changes. 

Other stuff can be added incrementally. I really want to flesh out the conceptual  documentation but that will take time and shouldn't delay a launch.

I want to change our blog post format to make them less technical and more show-and-tell so don't worry about samples.

I don't really want to go the RC route as it delays work on Drawing and Fonts and V2 was supposed to be a minor version originally.

**Reply by antonfirsov** [reply]
I assume you mean "Pixel Buffers"? The whole page is outdated currently:
https://docs.sixlabors.com/articles/imagesharp/pixelbuffers.html

Just like "Memory Management":
https://docs.sixlabors.com/articles/imagesharp/memorymanagement.html

Including or at lest referencing technical details in the blog post helps users with migration. The more information is given out proactively, the fewer basic questions land on the repository.

**Reply by JimBobSquarePants** [reply]
Yep, I updated the first replacing all the samples. I've deleted memory management for now, as the new manager shouldn't require customization for most users.

**Reply by antonfirsov** [reply]
Cool let me know if there is a PR, I keep missing docs repo changes.

**Comment by ronaldbarendse** [comment]
Will ImageSharp 2 also be built against .NET 6 (which is a LTS version)? The current alpha versions aren't, even though https://github.com/SixLabors/ImageSharp/pull/1745 added support, but that still seems to be behind a build flag:

https://github.com/SixLabors/ImageSharp/blob/cda6b24b63b33819bc2409a87d9be4323dcf2f41/src/ImageSharp/ImageSharp.csproj#L23-L44

**Reply by JimBobSquarePants** [reply]
No. There’s no reason to do so. We don’t use any .NET 6 specific APIs and consumers using .NET 6 will consume the .NET Core 3.1 target.

**Reply by JimBobSquarePants** [reply]
The build flag exists to allow us to isolate the build target for faster builds during CI.

**Reply by ronaldbarendse** [reply]
Doesn't that prevent you from using any of the .NET 5/6 compiler performance improvements though?

**Reply by JimBobSquarePants** [reply]
No. The distributed build uses the latest SDK installed on the VM. What you're looking at is targeted builds that are used explicitly for unit testing only.

**Comment by fydar** [comment]
**🎉 Congratulations on your 2.0 release! 🎉**

**Reply by JimBobSquarePants** [reply]
Thanks!!!

**Comment by sergiuciudin** [comment]
Good job, thank you all!
